### DISCUSSION TITLE
Add support for AmneziaWG WireGuard fork

### DISCUSSION THREAD

**Discussion by SNMetamorph** [pivpn/pivpn#1780] [Feature requests]
The core feature of this fork is protocol-level obfuscation techniques for bypassing deep packet inspection, that plain WireGuard protocol hasn't. But in other aspects it's similar to plain WireGuard and therefore it shouldn't be that hard to integrate support for this fork.

https://github.com/amnezia-vpn/amnezia-wg

**Comment by TrustCane** [comment]
I agree, it would be especially cool if it was possible to put on top of the already installed WG and the ability to switch modes (if such a thing is possible at all in theory)

**Comment by languagemaniac** [comment]
Did you get it to work?

I downloaded it, compiled it and runned amnezia-wg but it doesn't get recognized as a command.

Instead, when building, it builds wireguard-go.

I ran sudo wireguard-go awg0, and it doesn't show up in ifconfig but shows up if I execute "wg".

I'm unsure about how to proceed. The instructions on the GitHub are not very clear.

**Reply by marek22k** [reply]
You can use awg-quick. Otherwise, I think you can use any WireGuard tool with the prefix `a` - for example `awg` instead of `wg`.
